### ISSUE TITLE
Improvements to `new_tab` behaviour

### ISSUE THREAD

**Issue by peteryates** [x-govuk/govuk-components#489]
The `new_tab` argument on links adds the `target` and `rel` attributes to links so they open in a new tab, but it doesn't add the recommended text suffix `(opens in new tab)` to the link.

I think we should add the following behaviour:

* automatically append the text `(opens in new tab)` to links that have `new_tab: true`
* allow the text to be configured using the `default_link_new_tab_text` config option
* allow the behaviour to be suppressed by setting `default_link_new_tab_text` to `nil` or `false`
* add a new argument to link helpers called `visually_hidden_new_tab_text` or similar, which, if true, will append `<span class="govuk-visually-hidden">(opens in new tab)</span>` instead

**Comment by frankieroberto** [comment]
Hmm this one is tricky as ideally it wouldn’t be visually-hidden, but it can be if you have loads of links together - and I’m not sure how we’d detect that?

Perhaps we could default to adding it visibly to the link text unless either the link text already includes it, or the link text already has some visually-hidden text? 🤔

**Comment by peteryates** [comment]
Yeah I think visible by default, hidden via argument makes sense.

I'd probably lean towards showing a warning if it detects either "opens in" or any `.govuk-visually-hidden` content rather than affecting the output based on contents.
